spec/features/bulk_update_metadata_spec.rb

```diff
@@ -43,8 +43,11 @@
       next unless tds[3].text == note
 
       tds[9].find('form > button').click
-      # Confirm delete in the popup
-      click_link 'Delete'
+      within('#confirm-delete-modal') do
```

Comment:
Can you explain why we need this qualifier?

Reply:
It didn't work before; it works now. I can't offer a satisfying explanation.

Wondering if it has something to do with the recent changes to modals?

Reply:
possibly another delete button was added elsewhere on the page and the simple 'Delete' selector became ambiguous?

Reply:
@peetucket Capybara doesn't allow ambiguous selectors.